src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Themes/Generic.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:material="using:Uno.Material">
```

Comment:
No reason to have a generic.xaml in a sample, this is meant to be used in a library.

I believe it should only contain styles for controls we created (i.e. cards)